sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/SimpleAccessorNameNotationCheck.java

```diff
@@ -242,10 +241,7 @@ private static String getNameOfSettingField(DetailAST assign,
             else {
                 if (assigningFirstChild.getType() == TokenTypes.DOT) {
 
-                    if (assigningFirstChild.getChildCount() == 2
```

Comment:
Dot can only have 2 children, unless you can point me to an example that doesn't.